src/ambianic/pipeline/ai/fall_detect.py

```diff
@@ -243,11 +243,18 @@ def fall_detect(self, image=None):
                 pose_dix = {}
                 inference_result = []
 
-                for e_corr in self.fall_detect_corr:
-                    if pose.keypoints[e_corr].score > 0.6:
-                        # detected key-point probability must be greater than 0.6
-                        pose_dix[e_corr] = pose.keypoints[e_corr].yx
-                                        
+                # Find leftVectorScore and rightVectorScore --> i.e average score of shoulder & hip
+                leftVectorScore = (pose.keypoints['left shoulder'].score + pose.keypoints['left hip'].score) / 2.0
+                rightVectorScore = (pose.keypoints['right shoulder'].score + pose.keypoints['right hip'].score) / 2.0
+
+                if leftVectorScore > self.pose_confidence_threshold:
```

Comment:
This test should be replaced 

from:
```
  rightVectorScore = (pose.keypoints['right shoulder'].score + pose.keypoints['right hip'].score) / 2.0
```

to:

```
  rightVectorScore = min(pose.keypoints['right shoulder'].score, pose.keypoints['right hip'].score) 
```

same for the left vector score.

Reply:
Done

Reply:
Awesome. We just need a couple of test cases to cover the new formula code.